scripts/actions/mobskills/frozen_mist.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+--  Frozen Mist
+--  Description: Deals ice damage to enemies around the caster. Additional effect: Terror
+--  Type: Magical (Ice)
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.TERROR, 30, 0, 20)
+
+    local dmgmod = 2
```

Comment:
dmgmod should be 1.5 per Jimmayus retail testing here: https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=57955395

Duration is most likely 30 seconds base and the difference in duration is a resist state.

![image](https://github.com/LandSandBoat/server/assets/105882754/c3a8e04d-aaab-4f13-994c-6049aceb0387)

Reply:
Refactored the dmgmod. It looks like Terror is always 20 seconds at least from this capture video: https://www.youtube.com/watch?v=mbfAnJNJdoU. I tried testing with `applyResistanceAddEffect` but it always seems to return 1 so the duration ends up always being 30.

Reply:
Tested with more magic evasion and was able to get resist values, swapped to `xi.mobskills.applyPlayerResistance` as well for calculating resist.